plugin/grpc/grpcstats/client_metrics.go

```diff
@@ -97,7 +97,9 @@ func defaultClientMeasures() {
 }
 
 func defaultClientViews() {
-	RPCClientErrorCountView, _ = stats.NewView("grpc.io/client/error_count/cumulative", "RPC Errors", []tag.Key{keyOpStatus, keyService, keyMethod}, RPCClientErrorCount, aggCount, windowCumulative)
+	// Use the Java implementation as a reference at
+	// https://github.com/census-instrumentation/opencensus-java/blob/2b464864e3dd3f80e8e4c9dc72fccc225444a939/contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java#L113-L658
+	RPCClientErrorCountView, _ = stats.NewView(" rpc.io/client/error_count/cumulative", "RPC Errors", []tag.Key{keyOpStatus, keyService, keyMethod}, RPCClientErrorCount, aggMean, windowCumulative)
```

Comment:
rpc.io -> grpc.io

Reply:
Oops, nice catch @jcd2, I've fixed this up with https://github.com/census-instrumentation/opencensus-go/pull/225.